backend/src/main/java/com/example/backend/dto/WorkDto.java

```diff
@@ -1,4 +1,8 @@
 package com.example.backend.dto;
 
-public record WorkDto(String id, String carId, String type, int mileage, String date, double price) {
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.time.LocalDate;
+
+public record WorkDto(String id, String carId, String type, int mileage, @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd") LocalDate date, double price) {
```

Comment:
The only difference between WorkDto and CreateWorkRequest is the id field. Do you need both of them?

Reply:
I think so, since one dto is used to create an entity (which does not yet have an id), and the other is used to work with existing records.